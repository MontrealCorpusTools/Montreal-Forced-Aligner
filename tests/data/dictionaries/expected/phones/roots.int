shared split 1 2 3 4 5
shared split 6 7 8 9 10
shared split 11 12 13 14
shared split 15 16 17 18
shared split 19 20 21 22
