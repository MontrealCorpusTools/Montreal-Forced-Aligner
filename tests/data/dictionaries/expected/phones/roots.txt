shared split sil sil_B sil_E sil_I sil_S
shared split spn spn_B spn_E spn_I spn_S
shared split phonea_B phonea_E phonea_I phonea_S
shared split phoneb_B phoneb_E phoneb_I phoneb_S
shared split phonec_B phonec_E phonec_I phonec_S
